src/lib.rs

```diff
@@ -1486,6 +1503,8 @@ enum IoExpressionInner {
     EnvRemove(OsString),
     FullEnv(HashMap<OsString, OsString>),
     Unchecked,
+    #[cfg(unix)] Uid(u32),
+    #[cfg(unix)] Gid(u32),
```

Comment:
Ok, I played with this a bit. I think we can use `#[cfg_attr(windows, allow(dead_code))]` on just these two lines. That should let us get rid of all the other `#[cfg(unix)]` tags that are only there to suppress warnings. (Though we still need the one in `start_argv`, where we call methods that don't exist on Windows.)

Reply:
Note to future self / anyone else reading along: I'm able to test this from Unix with:

    rustup target add x86_64-pc-windows-gnu
    cargo check --target=x86_64-pc-windows-gnu

Rustup is *the best*.